src/Build/Evaluation/PropertyTrackingEvaluatorDataWrapper.cs

```diff
@@ -319,15 +332,17 @@ private void TrackPropertyReassignment(P? predecessor, P property, string? locat
 
             // Either we want to specifically track property reassignments
             // or we do not want to track nothing - in which case the prop reassignment is enabled by default.
-            if ((_settings & PropertyTrackingSetting.PropertyReassignment) == PropertyTrackingSetting.PropertyReassignment ||
-                (_settings == 0 && ChangeWaves.AreFeaturesEnabled(ChangeWaves.Wave17_10)))
```

Comment:
Not 100% sure I understand the removal of the changewave check here; can you elaborate?

Reply:
I should have changed it to Wave17_14 (due to Wave17_10 deprecation) instead of removal, fixed